packages/flutter_tools/test/general.shard/build_system/targets/android_test.dart

```diff
@@ -362,4 +364,152 @@ void main() {
       .childDirectory('arm64-v8a')
       .childFile('app.so').existsSync(), true);
   });
+
+  test('copyDeferredComponentSoFiles', () {
```

Comment:
If you're using the memory file system you shouldn't need `flush: true`, I believe that is mostly a no-op